Gemfile

```
@@ -10,9 +10,9 @@ group :development do
   gem 'stringex', '2.8.6'
   # > 2.1.0 causes slowdowns https://github.com/sass/sassc-ruby/issues/189
   gem 'sassc', '2.1.0'
-  gem 'sass-embedded', '1.87.0'
-  gem 'rubocop', '1.75.5'
-  gem 'ruby-lsp', '0.23.17'
+  gem 'sass-embedded', '1.89.0'
+  gem 'rubocop', '1.75.6'
```

RuboCop (1.75.5)

[convention] Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `rubocop` should appear before `sass-embedded`.

(Bundler/OrderedGems)